models/models.js

```diff
@@ -90,3 +95,26 @@ exports.fetchUsers = () => {
     return users;
   });
 };
+
+exports.deleteComment = (comment_id) => {
+  return db
+    .query(`DELETE FROM comments WHERE comment_id = $1 RETURNING *;`, [
+      comment_id,
+    ])
+    .then(({ rows }) => {
+      return rows;
+    });
+};
+
+exports.fetchCommentsByID = (comment_id) => {
+  const deleteQuery = `SELECT * FROM comments WHERE comment_id = $1`;
+  return db.query(deleteQuery, [comment_id]).then((result) => {
+    if (result.rows.length === 0) {
+      return Promise.reject({
+        status: 404,
+        msg: `Comment ${comment_id} was not found`,
+      });
+    }
+    return result.rows;
+  });
```

Comment:
we don't necessarily need this function here as if the comment_id doesn't exist it will get picked up in the deleteComment function and you will be able to handle the error from there